forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -558,7 +558,14 @@ public final int loseLife(int toLose, final boolean damage, final boolean manaBu
             final Map<AbilityKey, Object> repParams = AbilityKey.mapFromAffected(this);
             repParams.put(AbilityKey.Result, oldLife-toLose);
             repParams.put(AbilityKey.IsDamage, damage);
+            repParams.put(AbilityKey.LifeAmount, toLose);
 
+            switch (getGame().getReplacementHandler().run(ReplacementType.LoseLife, repParams)) {
+                case NotReplaced:
+                    break;
+                case Updated:
+                    toLose = (int) repParams.get(AbilityKey.LifeAmount);
+            }
             switch (getGame().getReplacementHandler().run(ReplacementType.LifeReduced, repParams)) {
```

Comment:
wouldn't the Map Params be created a new for LifeReduced?

currently AbilityKey.Result might be wrong because it isn't updated after LoseLife